dlib/simd/simd8i_vec.h

```diff
@@ -0,0 +1,234 @@
+// Copyright (C) 2013  Davis E. King (davis@dlib.net)
```

Comment:
This whole file looks like a copy of simd8i.h.  Can't we just #include simd8i.h?